core/src/main/web/app/helpers/menupluginmanager.js

```diff
@@ -138,6 +141,9 @@
           });
         }
       }
+      if (bkUtils.isElectron && menuChanged){
+        bkElectron.updateMenus(menus);
```

Comment:
It applies all changes done to the non-native menu bar (not shown on Electron window) to the electron native menu.